src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,44 @@
+package duke.tasks;
+
+import java.util.Queue;
+
+public class Deadline extends Task {
+
+    // toString format string
+    private static final String FORMAT_STRING = "[%c][%c] %s (by: %s)";
+
+    protected String byDate;
+
+    /**
+     * Create a Deadline with isDone set to false, description of choice and byDate of choice
+     *
+     * @param description Description of the Task to be created
+     * @param byDate String representing deadline
+     */
```

Comment:
In method header comments, the first sentence should start in the form "Creates ..." instead of "Create ...".